x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -1047,6 +1047,94 @@
             }
           }
         },
+        "count_rules_executions_timeouts_per_day": {
+          "type": "long"
+        },
+        "count_rules_executions_timeouts_by_type_per_day": {
+          "properties": {
+            "DYNAMIC_KEY": {
+              "type": "long"
+            },
+            "__index-threshold": {
+              "type": "long"
+            },
+            "__es-query": {
+              "type": "long"
+            },
+            "transform_health": {
+              "type": "long"
+            },
+            "apm__error_rate": {
+              "type": "long"
+            },
+            "apm__transaction_error_rate": {
+              "type": "long"
+            },
+            "apm__transaction_duration": {
+              "type": "long"
+            },
+            "apm__transaction_duration_anomaly": {
+              "type": "long"
+            },
+            "metrics__alert__threshold": {
+              "type": "long"
+            },
+            "metrics__alert__inventory__threshold": {
+              "type": "long"
+            },
+            "logs__alert__document__count": {
+              "type": "long"
+            },
+            "monitoring_alert_cluster_health": {
+              "type": "long"
+            },
+            "monitoring_alert_cpu_usage": {
+              "type": "long"
+            },
+            "monitoring_alert_disk_usage": {
+              "type": "long"
+            },
+            "monitoring_alert_elasticsearch_version_mismatch": {
+              "type": "long"
+            },
+            "monitoring_alert_kibana_version_mismatch": {
+              "type": "long"
+            },
+            "monitoring_alert_license_expiration": {
+              "type": "long"
+            },
+            "monitoring_alert_logstash_version_mismatch": {
+              "type": "long"
+            },
+            "monitoring_alert_nodes_changed": {
+              "type": "long"
+            },
+            "siem__signals": {
```

Comment:
There appear to be 6 SIEM rule types now: https://github.com/elastic/kibana/blob/1ecb42bf010bdd971f92794c3abe2ee47e4319c2/x-pack/plugins/security_solution/server/plugin.ts#L239-L246

Is this list managed manually?

Reply:
Yes, unfortunately....I will update. Thanks!

Reply:
Added in https://github.com/elastic/kibana/pull/123291/commits/b2c73b91701226e5d7ff1b2a73951d5e74287717